README.md

```diff
@@ -6,16 +6,18 @@
 ![Black status badge](https://github.com/Hochfrequenz/python-generics/workflows/Formatting/badge.svg)
 
 This package provides functionalities to determine the values of generic type variables in Python.
-As of now, it only supports two functions: `get_type_vars` and `get_filled_type`. These functions work also
-with pydantic generic models (only tested with pydantic > v2.3.0).
+As of now, it only supports two functions: `get_type_vars` and `get_filled_type`.
+These functions work also with pydantic generic models (only tested with pydantic > v2.3.0).
+
+The package has no dependencies itself.
 
 ### Installation
 The package is [available on PyPI](https://pypi.org/project/python-generics/):
 ```bash
 pip install python-generics
 ```
 
-### Usage
+### How to use (in our own application)
```

Comment:
Warum `in our own application`? Das ist durchaus allgemeingültig.

Reply:
da fehlte ein "y": https://github.com/Hochfrequenz/python-generics/pull/6/commits/8da3090491cc5a20582405b36038dde3132882aa